app/views/users/phone_setup/spam_protection.html.erb

```diff
@@ -0,0 +1,45 @@
+<% title t('titles.spam_protection') %>
+
+<%= image_tag asset_url('2FA-voice.svg'), alt: '', width: 200, height: 113, class: 'margin-bottom-2' %>
+
+<%= render PageHeadingComponent.new.with_content(t('titles.spam_protection')) %>
+
+<p><%= t('forms.spam_protection.description') %></p>
+
+<%= simple_form_for(@new_phone_form, url: request.url, method: :post) do |f| %>
+  <%= f.input :phone, as: :hidden %>
+  <%= f.input :international_code, as: :hidden %>
+  <%= f.input :otp_delivery_preference, as: :hidden %>
+  <%= f.input :otp_make_default_number, as: :hidden %>
+  <%= f.input :recaptcha_version, as: :hidden, input_html: { value: 2 } %>
```

Comment:
is this fallback page always version 2? is there a way to set this fro a controller or local variable from the form? just to minimize the number of places we hardcode things?

Reply:
Yes it is. Although interestingly, Google enforces that the reCAPTCHA keys matches the configured version for the frontend display of the checkbox, but it appears that any reCAPTCHA key can be used for the verification endpoint.

There might be some option to configure this in the controller. One interesting thing is that all of these values will carry over from the previous submission, _including_ `recaptcha_version: 3` from the initial challenge. We'd want to make sure that we set that at the right point to make sure that we'd not be populating the wrong version in the form input.

Reply:
I think that alternative ends up looking like adding a line before this:

https://github.com/18F/identity-idp/blob/38e3643026574229cda5b230543e927954c3a9c4/app/controllers/users/phones_controller.rb#L23

...as:

```rb
@new_phone_form.recaptcha_version = 2
```

...but it feels a little awkward / unpredictable to modify the form like that ahead of rendering the view? I guess contrasted with how it's implemented here, I'm a bit more comfortable with forcefully overriding the value in the view.